src/main/webapp/script.js

```diff
@@ -81,13 +94,29 @@ async function getChart() {
 
     const chart = new google.visualization.ColumnChart(
         document.getElementById('chart-container'));
-    chart.draw(view, options);    
-
-    const averageContainer = document.getElementById('average-score-container');
-    averageContainer.innerHTML = "Average Sentiment Score: " + averageScore;
-  });
+    chart.draw(view, options); 
 }
 
+/**
+ * Create a word cloud based on the number of appearance for each word
+ * @param {Map<String:Integer>} wordFrequencyMap 
+ */
+function getWordCloudChart(wordFrequencyMap) {
+    var data = [];
+    Object.keys(wordFrequencyMap).forEach(wordKey => data.push({"x": wordKey, "value": wordFrequencyMap[wordKey]}));
+    // create a tag cloud chart
```

Comment:
Capitalize the first letter of the comments to be consistent with the rest of the files.